tests/fixtures-esm/v2-api-with-path/with-named-group.mjs

```diff
@@ -0,0 +1,10 @@
+export default async () =>
+  new Response('<h1>Hello world</h1>', {
+    headers: {
+      'content-type': 'text/html',
+    },
+  })
+
+export const config = {
+  path: '/products/:id',
```

Comment:
do we want to add a test for multiple named groups as well?

like `/products/:category/:variant/:sku`

Reply:
Great idea. Done in https://github.com/netlify/zip-it-and-ship-it/pull/1523/commits/b7fbde318debce338ca3a4a1193ece0226ca6271.